main.go

```diff
@@ -0,0 +1,209 @@
+package main
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"net/http/cookiejar"
+	"os"
+	"time"
+
+	"github.com/xuri/excelize/v2"
+)
+
+const (
+	thirtyDays  int = 30
```

Comment:
maybe name this `maxPasswordAgeDays` or something else that expresses its meaning a little more clearly?

Reply:
Done. Much better.